roadmaps/configuration.md

```diff
@@ -85,27 +87,34 @@ they can build on**
 * Because of remote execution, this might not be the same machine Bazel runs on
 
 
-<div class="etabox">Aug 2018</div>**C++ rules fully support
+<div class="etabox">Dec 2018</div>**C++ rules fully support
 [platforms](https://docs.bazel.build/versions/master/platforms.html) and
 [toolchains](https://docs.bazel.build/versions/master/toolchains.html)**
 <span class="inprogressstatus">IN PROGRESS</span>
 
-* This gives them first-class Skylark support, `select()` [on
+* This gives them first-class Starlark support, `select()` [on
 platforms](https://docs.bazel.build/versions/master/be/general.html#config_setting.constraint_values),
 and configuration via
 [--platforms](https://docs.bazel.build/versions/master/platforms.html#specifying-a-platform-for-a-build)
 * These set best practice templates for adoption by other rules
 
 
-<div class="etabox">Sep 2018</div>**There's _one_, simple way to choose platforms
+<div class="etabox">Dec 2018</div>**"Toolchain modes" support same-platform
```

Comment:
Current plans for Toolchain Modes is that it's not a new feature, it's a way to write toolchains using flags (from, say, starlark build configuration) to properly select. I will be writing documentation for this, so possibly reword to:

**"Toolchain modes" documentation explains how to use flags and configuration to select between multiple toolchains for the same platform.

Reply:
I think it's still fair to call this a deliverable feature, even if (IIRC) it doesn't now require new technical refactoring?  Certainly from a user perspective I think it's a distinct & clearly valued concept.

Reply:
We should track the documentation on the roadmap, sure. See my proposed wording in the previous comment.

Reply:
Ack done.